src/main/java/axiomq/com/starwars/entities/Character.java

```diff
@@ -0,0 +1,48 @@
+package axiomq.com.starwars.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class Character {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
If you are using MySql while testing app with POSTMAN, maybe it's better to use GenerationType.IDENTITY